internal/services/azuremanagedlustrefilesystem/managed_lustre_file_system_resource.go

```diff
@@ -103,25 +136,17 @@ func (r ManagedLustreFileSystemResource) Arguments() map[string]*pluginsdk.Schem
 		},
 
 		"sku_name": {
-			Type:     pluginsdk.TypeString,
-			Required: true,
-			ForceNew: true,
-			ValidateFunc: validation.StringInSlice([]string{
-				"AMLFS-Durable-Premium-40",
-				"AMLFS-Durable-Premium-125",
-				"AMLFS-Durable-Premium-250",
-				"AMLFS-Durable-Premium-500",
-			}, false),
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(PossibleValuesForSkuName(), false),
 		},
 
 		"storage_capacity_in_tb": {
-			Type:     pluginsdk.TypeInt,
-			Required: true,
-			ForceNew: true,
-			ValidateFunc: validation.All(
-				validation.IntBetween(8, 128),
-				validation.IntDivisibleBy(8),
```

Comment:
this pr removes the increment check?

Reply:
No, that is done in the `CustomizeDiff` now, this removes the upper limit for the storage capacity as many customers have had exceptions granted to their subscriptions for this resource.